spec/sms_handler_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'sms_handler'
+
+describe SmsHandler do
+  subject(:sms_handler) { described_class.new }
+  let(:client) { double(:client) }
+  let(:message) { double(:message) }
+
+  describe '#send_sms' do
+    it 'makes a new client' do
+      allow(client).to receive_message_chain(:messages, :create)
+      expect(Twilio::REST::Client).to receive(:new)
+        .with(ENV['ACCOUNT_SID'], ENV['AUTH_TOKEN'])
+        .and_return(client)
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver. (https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)